tools/install_prereqs.sh

```diff
@@ -56,8 +56,14 @@ echo "Dependencies are installed at $PREFIX"
 # Install deps with rbuild
 rbuild prepare -d $PREFIX -s develop
 
-# install onnx package for unit tests
-pip3 install onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
+if python3 -c 'import sys; assert sys.version_info >= (3, 10)' > /dev/null; then
+  export CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
+  python3.10 -m pip onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
```

Comment:
```suggestion
  python3 -m pip onnx==1.10.2 numpy==1.21.6 typing==3.7.4 pytest==6.0.1 packaging==23.0
```
It should fall back to `python3.10` automatically if `if` condition using `python3` was run and true. 

But what was the issue that required a separate install for python3.10?  

Is that multiple pythons versions are in 22.04 ?

Reply:
Yeah there is, the default is 3.10 which needs these installs and flags to have make check working correctly